tdrs-backend/tdpservice/data_files/views.py

```diff
@@ -55,6 +55,12 @@ def filter_queryset(self, queryset):
             self.filterset_class = None
         return super().filter_queryset(queryset)
 
+    def get_serializer_context(self):
+        """Retrieve additional context required by serializer."""
+        context = super().get_serializer_context()
+        context['user'] = self.request.user
```

Comment:
This is what gives the `validate_file` method on the serializer access to the user.